statsmodels/stats/_lilliefors.py

```diff
@@ -216,21 +308,37 @@ def kstest_normal(x, pvalmethod='approx'):
     '''
 
     x = np.asarray(x)
-    z = (x-x.mean())/x.std(ddof=1)
-    nobs = len(z)
+    nobs = len(x)
+
+    if dist == 'norm':
+        z = (x-x.mean())/x.std(ddof=1)
+        test_d = stats.norm.cdf
+        lilliefors_table = get_lilliefors_table(dist='norm')
+    elif dist == 'exp':
+        z = x
+        ex = stats.expon(scale=1./x.mean())
```

Comment:
Ah! You're correct, my scale should be inverted! Nice catch.  
Lilliefors calls for `lambda = 1/X.mean()`

Then `scale = 1/lamda` for the usual parameterization. 

```
A common parameterization for expon is in terms of the rate parameter lambda, such that pdf = lambda * exp(-lambda * x). This parameterization corresponds to using scale = 1 / lambda.
```
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.expon.html
 
I'll fix it.

That said, I don't believe scaling `x = x/x.mean()` by the mean is the equivalent to setting `lambda=x.mean()`.

Try the below as a demonstration. You'll get different distributions.

```
ex_dist = stats.expon(scale=1/0.5)
X_scaled = ex_dist.rvs(size=10000)
X_norm = X_scaled / X_scaled.mean()
X_estim = stats.expon(scale=1/(1./X_scaled.mean())).rvs(size=10000)
sns.distplot(X_norm)
sns.distplot(X_estim)
```

Here's the resulting plot.

![figure_1](https://user-images.githubusercontent.com/2465221/28698189-786b1ffa-72f6-11e7-8d5b-244ccb2c87b0.png)

Does that seem reasonable, or am I missing something here?


Thanks for the helpful feedback!